modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+
+[id="installation-registry-storage-block-recreate-rollout_{context}"]
+= Configuring block registry storage for VMware vSphere
+
+To allow the image registry to use block storage types such as AWS Elastic Block Store (EBS) during upgrades as a cluster administrator, you can use the `Recreate` rollout strategy.
```

Comment:
The title is about `VMware vSphere`, but the text is about `AWS`. Should it be about vSphere VMDK?

Reply:
Excellent point. I've replaced `AWS EBS` with `vSphere VMDK`. 
@dmage Can you confirm that it is still accurate here to say "block storage types such as vSphere VMDK"?

Reply:
@bobfuru yes, it looks good